docs/index.rst

```diff
@@ -49,6 +49,12 @@ Information about the GMT data server and mirrors can be found on the
         :img-bottom: /_static/GMT_earth_faa_thumbnail.jpg
         :text-align: center
 
+    .. grid-item-card:: IGPP Earth Free-Air Anomaly Error
+        :link: earth-faaerrors
+        :link-type: doc
+        :img-bottom: /_static/GMT_earth_faaerrors_thumbnail.jpg
```

Comment:
We don't have this image yet.

Reply:
I just added what I think are all the missing images and added code to index.rst but getting some errors. Maybe you can see what is missing.

Reply:
Duh. Need to add all the earth-edefl.rst etc etc.  Can @Esteban82 help?